eth_account/_utils/encode_typed_data/helpers.py

```diff
@@ -0,0 +1,61 @@
+from typing import (
+    Any,
+)
+
+
+def _get_eip712_solidity_types():
+    types = ["bool", "address", "string", "bytes"]
```

Comment:
It seems that `V4` allows for `uint` and `int` types and `eth-sig-util` tests for these. Just by adding the types to this list, our implementation in this PR seems to pass all of those tests.

```suggestion
    types = ["bool", "address", "string", "bytes", "uint", "int"]
```

Reply:
added